regression/goto-instrument/restrict-function-pointer-to-multiple-functions-via-file-and-command-line-options/test.c

```diff
@@ -35,38 +19,19 @@ int g(int x)
 
 int h(int x)
 {
-  return x - 1;
-}
-
-int select_function = 0;
-
-void select_f(void)
-{
-  select_function = 0;
+  return x;
 }
 
-void select_g(void)
+int other(int x)
 {
-  select_function = 1;
+  return x;
 }
 
-void select_h(void)
+int main(void)
 {
-  select_function = 2;
+  use_f(f);
+  use_f(g);
+  use_f(h);
+  use_f(other);
 }
 
-fptr_t get_f(void)
```

Comment:
I guess all this deletion would be clearer if put in its own commit explaining why